tests/abstract-tensor-tests.lisp

```diff
@@ -4,6 +4,81 @@
 
 (in-package #:magicl-tests)
 
+(defmacro swapping-arguments-is ((predicate arg1 arg2))
```

Comment:
This can be a function if you'll tolerate a FUNCALL. `is` doesn't need to be lexically visible.